.github/stale.yml

```diff
@@ -0,0 +1,84 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 335
```

Comment:
This feels really long IMO, how about 3-6 months? Cleanup sooner also ensures we are picking and working on backlogs at some point. just a personal preference though? @jahtalab thoughts?

Reply:
Sure thing, I just took what we got in place as a template, happy to change it :)

Reply:
I agree, we should make a decision about an issue within 3-6 months (possibly even sooner than that but to reduce noise let's set it at 4 month). We either close the issue or add an appropriate label (e.g. `bug`, `enhancement`) to it.